Dockerfile

```diff
@@ -0,0 +1,40 @@
+
+# TODO: make this default to govuk-ruby once it's being pushed somewhere public
+# (unless we decide to use Bitnami instead)
+ARG base_image=ruby:2.7.2
```

Comment:
This is an example of a naughty app that's pinning an obsolete patch version of CRuby 😅 (Not something to worry about right now, just something we'll need to fix up soon-ish as part of getting our common base/build images into place. There are plenty of others with the same issue.)

Reply:
Yeah I just matched the version in .ruby-version to be safe